src/App.tsx

```diff
@@ -1,61 +1,137 @@
+import { Todo } from './interfaces/Todo';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
```

Comment:
[CODE STYLE] - Prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components. Here you are importing `usersFromServer` and `todosFromServer` directly in the App component, which is fine since it's the root component. However, make sure that any child components receive their necessary data through props and do not import such data themselves.